lib/web_ui/lib/src/engine/util.dart

```diff
@@ -627,6 +628,23 @@ extension JsonExtensions on Map<dynamic, dynamic> {
   }
 }
 
+/// Extracts view ID from the [MethodCall.arguments] map.
+///
+/// Throws if the view ID is not present or if [arguments] is not a map.
+int readFlutterViewId(Object? arguments) {
+  return tryFlutterViewId(arguments)!;
+}
+
+/// Extracts view ID from the [MethodCall.arguments] map.
+///
+/// Returns null if the view ID is not present or if [arguments] is not a map.
+int? tryFlutterViewId(Object? arguments) {
+  if (arguments is Map) {
+    return arguments.tryInt('flutterViewId');
```

Comment:
I'm trying to figure out if this is going to be named `'viewId'` or something else before we land this :P (for the `render`, for example)

Reply:
Yeah, the change is going to be a little bit of a pain, since the string needs to be kept in sync between the engine <-> framework. I think @goderbauer mentioned he'd use `viewId` for other messages. Should we rename this (and the fw version)?